packages/devtools_app/lib/src/timeline/flutter/timeline_service.dart

```diff
@@ -31,44 +34,29 @@ class TimelineService {
       _handleConnectionStart(serviceManager.service);
     }
     serviceManager.onConnectionClosed.listen(_handleConnectionStop);
+
+    timelineController.recording.addListener(() async {
+      await updateListeningState(true);
```

Comment:
what purpose is this `async` and `await` serving? 
`() => updateListeningState(true)`
should have the same impact.

Reply:
done